sdk/communication/Azure.Communication.Sms/tests/SessionRecords/SmsClientLiveTests/SendingAnSmsMessage.json

```diff
@@ -26,21 +23,21 @@
       "StatusCode": 200,
       "ResponseHeaders": {
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Tue, 10 Nov 2020 23:51:35 GMT",
-        "MS-CV": "ve2uWfjNcEiPGbYCNuRQEw.0",
+        "Date": "Tue, 19 Jan 2021 00:46:58 GMT",
+        "MS-CV": "4jMmke8A20ilhCqaGW98eQ.0",
+        "Request-Context": "appId=",
         "Transfer-Encoding": "chunked",
-        "X-Azure-Ref": "0hierXwAAAACfIn4t4LjSR4b9wyIW6Qt3WVRPMDFFREdFMDYxNgA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
-        "X-Processing-Time": "582ms"
+        "X-Azure-Ref": "0ASwGYAAAAAA3uvPg1AToSoYrV60svvs7WVZSMzBFREdFMDQwNgA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
+        "X-Processing-Time": "1051ms"
       },
       "ResponseBody": {
         "messageId": "Sanitized"
       }
     }
   ],
   "Variables": {
-    "AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING": "endpoint=https://sanitized.communication.azure.com/;accesskey=Kg==;",
-    "AZURE_PHONE_NUMBER": "\u002B18005551234",
-    "RandomSeed": "2069880153",
-    "TO_PHONE_NUMBER": "\u002B18005555555"
+    "AZURE_PHONE_NUMBER": "\u002B18005555555",
```

Comment:
Looks like this didn't get re-recorded for a while, because it still had TO_PHONE_NUMBER

Reply:
I did not re-record the managed identity tests.

Reply:
Good catch, I just recorded them again